templates/charon.yaml

```diff
@@ -41,6 +41,30 @@ spec:
       securityContext:
         runAsUser: 1000
         fsGroup: 1000
+      volumes:
+        - name: validators
```

Comment:
I think we can remove the validators keys from the charon template, charon only needs the cluster lock and the private key